### ISSUE TITLE
Accessibility for social media icons is broken

### ISSUE THREAD

**Issue by zersiax** [luizdepra/hugo-coder#126]
Currently, the social media icons don't offer any way for screen readers to infer where the links go. Can this be looked at? I amconsidering this theme for my blog/website but really need this to work first.

**Comment by luizdepra** [comment]
Sure, I can try. I think I can add `aria-label` and `aria-hidden` in the icons.
But, how can I test it?

**Comment by zersiax** [comment]
You can use the free, open-source NVDA screen reader or shoot me a copy of the altered code and I can give it a quick run-through.

**Comment by josephting** [comment]
[VoiceOver](http://www.apple.com/accessibility/voiceover/) (built-into macOS) can be used instead if you're on mac.

Another good help would be to use **Audits** tab in Chrome Developers' Tools. The accessibility audits provides insights on what should be done to optimize the page for better accessibility.

Source: [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Web_applications_and_ARIA_FAQ#How_do_I_test_my_use_of_ARIA_Are_there_any_tools_available_for_free)

**Comment by zersiax** [comment]
Very true :)
